lib/cli/commands/build.js

```diff
@@ -121,6 +121,17 @@ build.builder = function(cli) {
 			default: false,
 			type: "boolean"
 		})
+		.option("output-style", {
+			describe:
+				"Processes build results into a specific directory structure. " +
+				"\"Flat\" omits the project namespace and the \"resources\" directory. This is" +
+				" currently only supported for projects of type 'library'. Projects of " +
+				"type 'application' always result in a flat output." +
+				"No other dependencies can be included in the build result.",
+			type: "string",
+			default: "Default",
+			choices: ["Default", "Flat", "Namespace"],
```

Comment:
Indeed!

Reply:
With my question I meant to not enforce case at all (but still document the actual values to be aligned with cache-mode).
https://yargs.js.org/docs/#api-reference-coercekey-fn might be used for that.

Or we use (and document) all-lowercase and also change the cache-mode to be consistent again, but that would be breaking unless we still handle the existing case.

If we end up having mixed behaviour for different options I feel like it will be even worse for users.

@RandomByte what do you think?

Reply:
I agree that we should probably stick to capitalized keywords to be consistent with `cache-mode`. Allowing lowercase makes sense for both arguments, if it is easy to implement.

Reply:
WIth https://yargs.js.org/docs/#api-reference-coercekey-fn its fairly easy to allow input that ignores the case.
So, I suppose it should be ok to leave the capitalized keywords, but implement `coercekey` and "fix" the input if needed